content/02_general-rules/04_asset-modifications.md

```diff
@@ -1,10 +1,63 @@
 ### Asset Modifications
 
-Texture modifications are allowed, with some caveats. The textures of portal surfaces
-must not be modified in any way. No textures can be modified to provide any visual
-assistance, such as help with a lineup. No texture may be modified to be more
-translucent than it was, with the exception of the player's viewmodel.
-
-Modifying models is not allowed, with the exception of the portal gun (including both
-the viewmodel and the in-world model). Modifying audio files, captions or any other
-asset is not allowed.
+The asset modification rules will be split into their respective categories that are 
+seen when looking at the main pak01_dir.vpk in the game’s files. There will also be 
+a “General Rules” category that applies to any kind of asset modification.
+
+#### General Rules
+
+Any asset modifications are subject to being grounds for the rejection or removal of 
+any run on speedrun.com or the cm boards.
+
+When assets are required to be submitted alongside a run, these assets must adhere to 
+speedrun.com TOS and any other applicable TOS for wherever your proof of the run is 
+being stored. Including video submissions.
+
+Editing .dll files that are present in the files to modify an asset is not allowed.
```

Comment:
Suggest edit: "Editing binary files (including .DLL, .EXE, .SO, A.OUT, .DYLIB, or .APP) files which are present in the game or Steam directory..."

Although, shouldn't we really just state somewhere that this is blanket disallowed? I thought we already did, but I could be wrong.